prog/remove_boot_image.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Prog::RemoveBootImage < Prog::Base
+  subject_is :boot_image
+
+  label def start
+    boot_image.update(activated_at: nil)
+    hop_wait_volumes
+  end
+
+  label def wait_volumes
+    nap 30 if boot_image.vm_storage_volumes.count > 0
+    hop_remove
+  end
+
+  label def remove
+    # YYY: version will be enforced in future.
+    image_path =
+      boot_image.version ?
+          "/var/storage/images/#{boot_image.name}-#{boot_image.version}.raw" :
+          "/var/storage/images/#{boot_image.name}.raw"
+
+    boot_image.vm_host.sshable.cmd("sudo rm -rf #{image_path.shellescape}")
```

Comment:
When we download a new boot image, we decreases the available_storage_gib. Should we increase it when we remove the image?